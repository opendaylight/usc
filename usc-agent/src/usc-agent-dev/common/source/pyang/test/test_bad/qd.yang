submodule qd {
  belongs-to qa {
    prefix a;
  }

  include qc;

  leaf dl {
    type bt; // error; qb not included
  }
  container dc {
    if-feature bf; // error; qb not included
    uses bg; // error; qb not included
    leaf dl {
      type identityref {
        base bi; // error; qb not included
      }
    }
  }
  augment /bc { // error; qb not included
    leaf dl {
      type leafref {
        path "/bl"; // error; qb not included
      }
    }
  }
  

}
