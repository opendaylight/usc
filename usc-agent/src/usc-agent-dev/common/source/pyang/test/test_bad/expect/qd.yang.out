qd.yang:9: error: TYPE_NOT_FOUND
qd.yang:12: error: FEATURE_NOT_FOUND
qd.yang:13: error: GROUPING_NOT_FOUND
qd.yang:16: error: IDENTITY_NOT_FOUND
qd.yang:20: error: NODE_NOT_FOUND
qd.yang:23: error: LEAFREF_IDENTIFIER_NOT_FOUND
